### DISCUSSION TITLE
Where is Half-life ?

### DISCUSSION THREAD

**Discussion by Microbiotman** [Open-Systems-Pharmacology/Forum#1483] [Questions & Problems] [answered]
Hello everybody
When we run a simulation in PK-Sim, we obtain the Half-life **associated with terminal slope**. But I would like the **basic Half-life (50% of dose eliminated)** . Where can I find it ? 

Thank you for your help ! :)

**Comment by StephanSchaller** [comment, marked as answer]
Are you actually looking for 50% of the dose? There is no such function in PK-Sim.
Or 50% of Cmax?

**Reply by Microbiotman** [reply]
Hello Stephan,

I was looking for t1/2 50% of the dose..

Thank you for your help !
